crates/admin/src/schema_registry/error.rs

```diff
@@ -135,12 +135,12 @@ pub enum SubscriptionError {
 
 #[derive(Debug, thiserror::Error, codederror::CodedError)]
 pub enum DeploymentError {
-    #[error("existing deployment id is different from requested (requested = {requested}, existing = {existing})")]
-    #[code(restate_errors::META0004)]
-    IncorrectId {
-        requested: DeploymentId,
-        existing: DeploymentId,
-    },
+    #[error("an update deployment operation must provide an endpoint with the same services and handlers. The update tried to remove the services {0:?}")]
+    #[code(restate_errors::META0006)]
```

Comment:
I would define a new error code for errors related to update. META0006 seems wrong as it talks about adding deployments. In the error code you can describe in a more lengthy manner what to do